engine/engine_test.go

```diff
@@ -4740,24 +4759,29 @@ func TestNativeHistograms(t *testing.T) {
 		},
 	}
 
+	defer pprof.StopCPUProfile()
```

Comment:
this still needed? we could just run with "--cpuprofile test.pprof" instead i think